Lemon Effect Lua Error

Occasionally on my server when someone spawns/removes a lemon, this occurs

```
[ERROR] lua/effects/lemon_load.lua:10: Tried to use a NULL entity!
  1. GetPos - [C]:-1
   2. unknown - lua/effects/lemon_load.lua:10
```

--- a/lua/effects/lemon_load.lua
+++ b/lua/effects/lemon_load.lua
@@ -7,6 +7,10 @@ function EFFECT:Init( Input )
 	
 	local Entity = Input:GetEntity( )
 	
+	if !IsValid( Entity ) then
+		return false
+	end
+	
 	self:SetPos( Entity:GetPos( ) )
 	
 	self:SetAngles( Entity:GetAngles( ) + Angle( 0.01, 0.01, 0.01 ) )

